### PR TITLE
directory for mount volumes

### PR DESCRIPTION
**Pull request by giggsoff**
The ability to use local directory to create docker images in the local registry and use them as volume or as rootfs of app

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: docs/applications.md [hunk 1]
```diff
@@ -8,35 +8,51 @@ In order to deploy application you can use `eden pod deploy` command:
...
+      --config string      Name of config (default "default")
+  -v, --verbosity string   Log level (debug, info, warn, error, fatal, panic (default "info")
 ```
 
+### Mount
+
+You can pass additional volumes to application with `--volume` flag. For example:
+
+```bash
+eden pod deploy docker://itmoeve/eclient:0.7 -p 8027:22 --volume=/tst:docker://nginx:1.20.0 --volume=/dir:directory://./tests
```
Comment (mydatascience) [inline docs/applications.md:50]: may we support the docker syntax
 -v nginx-vol:/usr/share/nginx/html
for us will be: 
--volume docker://nginx-vol:/usr/share/nginx/html
or 
--volume /my/nginx:/usr/share/nginx/html

So if we don't have a prefix we assume it's a local dir

Comment (mydatascience) [reply docs/applications.md:50]: we may also wanna support some existing volumes?

Comment (giggsoff) [reply docs/applications.md:50]: Sorry, can you please elaborate about proposed existing volumes?

Comment (mydatascience) [reply docs/applications.md:50]: I mean we have the idea of volumes in Zcloud. You create it and then can list and mount.

**Pull request merged**